source/security.txt

```diff
@@ -4,8 +4,218 @@
+the relevant values for your MongoDB deployment.
+
+.. _php-auth-sample:
+
+.. include:: /includes/usage-examples/sample-app-intro.rst
+
+.. literalinclude:: /includes/usage-examples/connect-sample-app.php
+   :language: php
+   :copyable: true
+   :linenos:
+   :emphasize-lines: 5-7
+
+SCRAM-SHA-256
+-------------
+
+The following code shows how to authenticate by using the ``SCRAM-SHA-256``
+authentication mechanism:
+
+.. tabs::
+
+   .. tab:: MongoDB\\Client
+      :tabid: Client
+
+      .. literalinclude:: /includes/authentication.php
+         :language: php
+         :dedent:
+         :start-after: start-scram-sha-256-client
+         :end-before: end-scram-sha-256-client
+
+   .. tab:: Connection URI
+      :tabid: connectionstring
+
+      .. literalinclude:: /includes/authentication.php
+         :language: php
+         :dedent:
+         :start-after: start-scram-sha-256-uri
+         :end-before: end-scram-sha-256-uri
+
+To learn more about SCRAM-SHA-256 authentication, see :ref:`php-scram-sha-256` in
+the Authentication guide.
+
+SCRAM-SHA-1
+-----------
+
+The following code shows how to authenticate by using the ``SCRAM-SHA-1``
+authentication mechanism:
+
+.. tabs::
+
+   .. tab:: MongoDB\\Client
+      :tabid: Client
+
+      .. literalinclude:: /includes/authentication.php
+         :language: php
+         :dedent:
+         :start-after: start-scram-sha-1-client
+         :end-before: end-scram-sha-1-client
+
+   .. tab:: Connection URI
+      :tabid: connectionstring
+
+      .. literalinclude:: /includes/authentication.php
+         :language: php
+         :dedent:
+         :start-after: start-scram-sha-1-uri
+         :end-before: end-scram-sha-1-uri
+
+To learn more about SCRAM-SHA-1 authentication, see :ref:`php-scram-sha-1` in
+the Authentication guide.
+
+MONGODB X.509
+-------------
+
+The following code shows how to create a connection URI to authenticate by using
+the ``X.509`` authentication mechanism:
+
+.. tabs::
+
+   .. tab:: MongoDB\\Client
+      :tabid: Client
+
+      .. literalinclude:: /includes/authentication.php
+         :language: php
+         :dedent:
+         :start-after: start-mongodb-X509-client
+         :end-before: end-mongodb-X509-client
+
+   .. tab:: Connection URI
+      :tabid: connectionstring
+
+      .. literalinclude:: /includes/authentication.php
+         :language: php
+         :dedent:
+         :start-after: start-mongodb-X509-uri
+         :end-before: end-mongodb-X509-uri
+
+To learn more about X.509 authentication, see :ref:`php-x509` in
+the Authentication guide.
+
+MONGODB-AWS
+-----------
+
+The following sections show how to connect to MongoDB by using the ``MONGODB-AWS``
+authentication mechanism. When you use the ``MONGODB-AWS`` mechanism, the {+php-library+}
+attempts to retrieve your AWS credentials from the following sources, in the order listed:
+
+1. Options parameter passed to the ``MongoDB\Client`` constructor or parameters in the
+   connection URI
```

Comment:
See: https://github.com/mongodb/docs-php-library/pull/139/files#r1775267960

I would lead with "connection string" (or URI) and _then_ mention the `$uriOptions` array.